app/src/main/java/com/ncnf/utilities/map/MapHandler.java

```diff
@@ -122,6 +122,10 @@ public void update_markers(){
         clusterManager.cluster();
     }
 
+    /**
+     * Add the marker on the map for each event in the list (either its own marker or marker for a cluster of event)
+     * @param events List of events we want to add an marker
+     */
     private void addEventMarkers(List<Event> events){
```

Comment:
was it necessary to add doc to a private method?

Reply:
Just to make sure that people understand what it does even if the method is private yes